addons/web/static/src/views/graph/graph_view.scss

```diff
@@ -1,83 +1,7 @@
-.o_graph_view > .o_content > .o_renderer {
-  height: 100%;
-
-  .o_graph_canvas_container {
-    padding-top: 5px;
-    position: relative;
-    height: 100%;
-    width: 100%;
-    top: 0px;
-    left: 0px;
+.o_graph_view > .o_content > .o_renderer .o_graph_canvas_container {
+    min-height: 250px;
```

Comment:
I really would like to get rid of this ^^
Why is it necessary?

Reply:
@stefanorigano I would also like to 🙈 , but without the `min-height`, the graph can't be readable when the height of the viewport is too tiny. With this `min-height`, the canvas is still readable with a scroll bar at least.

![Capture d’écran 2022-01-26 à 15 56 54](https://user-images.githubusercontent.com/80679690/151188081-b023cb82-2727-4090-a40f-9050bfed588e.png)

With the API of BS5 we could create some utils classes to manage the min-height.